### ISSUE TITLE
Issue in preparing data for training

### ISSUE THREAD

**Issue by smahajan07** [twtygqyy/pytorch-LapSRN#2]
Hi!

I am preparing my own data for training. 
I am running the "generate_train.m" script from [pytorch-vdsr](https://github.com/twtygqyy/pytorch-vdsr/tree/master/data) and I have changed the path to the folder with images (approx 70k images). I have 2 doubts:
1. My images are of different dimensions, so will that cause a problem in training?
2. How do I add labels 'label_x2' and 'label_x4'?

I am to new to HDF format and hence the issue.

**Comment by twtygqyy** [comment]
@smahajan07 sry for the late reply. For LapSRN，you need to generate the hdf5 file with two outputs. I will push the code in the following days, basically just simple modification of "generate_train.m" script from pytorch-vdsr

**Comment by smahajan07** [comment]
@twtygqyy I have now managed to write a similar script in python. It seems to be working for now, but I am still facing some minor issue with the dimensions of images ; the patches of images it sends in the data and label part. The paper mentions that we send random 64 patches of 128x128, but in 'generate_train.m' we send all patches of the image, right?

PS. will wait for your push and will then compare my python script with it / or matlab, if that works fine on my end.
